README.md

```diff
@@ -155,7 +156,7 @@ The summary outlines:
 - permissionDeniedErrors: the number of directories or files that could not be read due to permissions
 - pathErrors: the number of paths where an unexpected error occurred while trying to identify bad log4j versions
 - pathsSkipped: the numbers of paths skipped from full identification of bad log4j versions due to the config options set
-- numImpactedFiles: the total number of results previously output
+- findings: the total number of findings previously output
```

Comment:
Is this change of field going to be an issue?
I can separately report `numImpactedFiles` and `findings` if preferred.

Reply:
Yes, I think we should report separately. The number of impacted files is a better measure of work needing to be done, while findings just reflects how much there might be even if duplicated within the same archive.

Reply:
Cool, I've made that change.